tests/kernel/install_ltp.pm

```diff
@@ -521,6 +521,10 @@ will be used.
 
 Overrides the official LTP GitHub repository URL.
 
+=head2 LTP_INSTALL_REBOOT
+
```

Comment:
This is just adding doc to existing feature, right?

Reply:
Yes.